tests/lib/src/common_utils.h

```diff
@@ -9,105 +9,182 @@
 using namespace InferenceEngine;
 
 /**
- * @brief Determine if InferenceEngine blob means image or not
+ * @brief Determine if InferenceEngine blob means image or not (OV API 1.0)
  */
 template<typename T>
 static bool isImage(const T &blob) {
-  auto descriptor = blob->getTensorDesc();
-  if (descriptor.getLayout() != InferenceEngine::NCHW) {
-    return false;
-  }
-  auto channels = descriptor.getDims()[1];
-  return channels == 3;
+    auto descriptor = blob->getTensorDesc();
+    if (descriptor.getLayout() != InferenceEngine::NCHW) {
+        return false;
+    }
+    auto channels = descriptor.getDims()[1];
+    return channels == 3;
+}
+
+/**
+ * @brief Determine if InferenceEngine blob means image or not (OV API 2.0)
+ */
+static bool isImage(const ov::Output<ov::Node> &input) {
+    const auto &layout = ov::layout::get_layout(input);
+    if (ov::layout::has_height(layout) && ov::layout::has_width(layout) && ov::layout::has_channels(layout)) {
```

Comment:
&& shape[ov::layout::channels_idx(layout)] == 3  ?

Reply:
Theoretically there can be CMYK images as inputs, but for consistency with isImage for OV API 1 added this condition